README.md

```diff
@@ -123,6 +123,36 @@ pkg remove -y javavmwrapper
 pkg remove -y java-zoneinfo
   ```
 
+### Compatibility upgrade to MONGODB 4.2 (for those using MongoDB 4.2 or having DB compatibility problems)
+
+The follow is the work around for upgrading MongoDB 3.6 to MonggoDB 4.2. This is to resolve conflict and crashes in Unifi Controller relating to MongoDB:
```

Comment:
Follow should be following and workaround is all one word, I believe.
Suggested change: "The following is a workaround for upgrading MongoDB 3.6 to MonggoDB 4.2 to resolve conflict and crashes in Unifi Controller related to MongoDB versions."